src/scss/components/_dialog.scss

```diff
@@ -33,17 +34,18 @@
 
   &__user {
     &__photo {
-      width: 30px;
-      height: 30px;
       float: left;
       border-radius: 50%;
       background-color: get-color(air);
       overflow: hidden;
       margin-right: 10px;
+
+      .picture__image {
+        float: left;
```

Comment:
@MarcoBrunoBR can we use flex on parent element? Flex could make the use of `wrap__dialog__itens` unnecessary